fastlane/spec/action_metadata_spec.rb

```diff
@@ -3,6 +3,10 @@
 describe Fastlane::Action do
   Fastlane::ActionsList.all_actions do |action, name|
     describe name do
+      before do
+        allow(FastlaneCore::FastlaneFolder).to receive(:path).and_return("./fastlane")
```

Comment:
@mgrebenets https://github.com/fastlane/fastlane/pull/8484 is merged, so you don't need these mock anymore😄 Could you rebase and update this PR?